Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/SORT__input_1920__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/SORT__input_1920__conf_.2//MOT17-02-FRCNN.txt


Processing tracker: LiteSORT, resolution: 2560, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/LiteSORT__input_2560__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/LiteSORT__input_2560__conf_.2//MOT17-11-FRCNN.txt


Processing tracker: DeepSORT, resolution: 2560, and confidence: .2
WARNING: Tracker folder /home/juma/code/StrongSORT/results/detectorParams/runs/DeepSORT__input_2560__conf_.2 does not exist. Skipping evaluation.
-------------------------------------------
Processing tracker: StrongSORT, resolution: 320, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 320, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 320, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 320, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_320__conf_.2//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 640, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 640, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 640, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 640, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_640__conf_.2//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 960, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 960, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 960, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 960, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_960__conf_.2//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1280, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1280, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1280, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1280, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1280__conf_.2//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1600, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1600, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1600, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1600, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1600__conf_.2//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1920, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1920, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1920, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 1920, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_1920__conf_.2//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2240, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2240, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2240, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2240, and confidence: .2
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.2
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2240__conf_.2//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2560, and confidence: .05
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2560__conf_.05
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2560__conf_.05//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2560, and confidence: .1
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2560__conf_.1
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2560__conf_.1//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2560, and confidence: .15
Running evaluation for the specified tracker folder: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2560__conf_.15
Traceback (most recent call last):
  File "TrackEval/scripts/run_mot_challenge.py", line 91, in <module>
    dataset_list = [trackeval.datasets.MotChallenge2DBox(dataset_config)]
  File "/home/juma/code/StrongSORT/TrackEval/trackeval/datasets/mot_challenge_2d_box.py", line 150, in __init__
    raise TrackEvalException(
trackeval.utils.TrackEvalException: Tracker file not found: /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2560__conf_.15//MOT17-02-FRCNN.txt
-------------------------------------------
Processing tracker: StrongSORT, resolution: 2560, and confidence: .2
WARNING: Tracker folder /home/juma/code/StrongSORT/results/detectorParams/runs/StrongSORT__input_2560__conf_.2 does not exist. Skipping evaluation.
Evaluation script completed.